src/App.css

```diff
@@ -603,19 +607,21 @@ a.text-link {
   top: 0.3em;
 }
 
-.child .editable,
-.editable-annotation-text {
+.child .editable {
   /* Add some padding-right for increased click area. */
   /* Add some padding-left otherwise caret is invisible on empty elements. */
   /* Cannot use padding-top on editable, as clicking it causes the selection to go to the beginning. */
   /* Cannot use padding-bottom on editable, as clicking it causes the selection to go to the end. */
   padding: 0 1em 0 0.333em;
+  box-sizing: border-box;
+}
+
+.child .editable,
+.editable-annotation-text {
   /* slide editable up so that it overlaps with the previous thought, ensuring there is no dead click zone from clip-path */
-  margin: -0.5px -1em 0 calc(1em - 18px);
+  margin: -0.5px 0 0 calc(1em - 18px);
   /* create stacking order to position above thought-annotation so that function background color does not mask thought */
   position: relative;
-  /* Prevent the selection from being incorrectly set to the beginning of the thought when the top edge is clicked, and the end of the thought when the bottom edge is clicked. Instead, we want the usual behavior of the selection being placed near the click. */
-  clip-path: inset(0.001px 0 0.1em 0);
```

Comment:
Was removing this intentional?

Reply:
Yes, I deleted it to view the annotation text for debugging purpose, but forgot to restore the change. 
I added them back.